go/vt/mysqlctl/mysql_daemon.go

```diff
@@ -53,6 +53,9 @@ type MysqlDaemon interface {
 	// GetServerUUID returns the servers UUID
 	GetServerUUID(ctx context.Context) (string, error)
 
+	// GetServerStatus returns the server statuses asked for.
+	GetServerStatus(ctx context.Context, statuses []string) ([]string, error)
```

Comment:
IMO this signature is clearer:
```
	// GetGlobalStatusVars returns the server's global status variables asked for.
        // An empty/nil variable name parameter slice means you want all of them.
	GetGlobalStatusVars(ctx context.Context, names []string) (map[string]string, error)
```

Reply:
The empty/nil part is still up for debate. Whether that should return none (as it does now) or all. In my mind, and aligning with existing MySQL behavior, I tend to think that providing no specific names/filters would return all. But I don't feel strongly about it. If you DO want to request all of them, doing so would be quite difficult otherwise.

The return value of a map IMO makes this much easier and more reliable to use. The order of variable names passed is not guaranteed to match the order of variable values returned. And even if it did, if you request e.g. 5 of them the caller has to track the index values of what was requested and what was returned.

Reply:
Done!